presto-cassandra/pom.xml

```diff
@@ -248,6 +262,9 @@
                     <ignoredNonTestScopedDependencies>
                         <ignoredNonTestScopedDependency>com.fasterxml.jackson.core:jackson-databind</ignoredNonTestScopedDependency>
                     </ignoredNonTestScopedDependencies>
+                    <ignoredUnusedDeclaredDependencies>
```

Comment:
> [!TIP]
> Code reviewers assist - This is to avoid `maven-check` failing from https://github.com/prestodb/presto/actions/runs/20494639197/job/58892495010?pr=26851#step:6:8689